modules/dkan/dkan_data_dictionary/css/dkan_data_dictionary.css

```diff
@@ -0,0 +1,3 @@
+#field-conformsto-schema-add-more-wrapper {
+  height: 400px;
```

Comment:
This style doesn't provide the intended results. See the attached screenshot. The json editor spills over the fieldset container: 
![screenshot of dkan](https://user-images.githubusercontent.com/21045418/36119682-fa500720-0ffd-11e8-894e-e05ce641e68a.jpg)

This can be fixed if the height value is applied as follows:

```
#field-conformsto-schema-add-more-wrapper .jsoneditor  {
  height: 400px;
}
```